kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -169,11 +194,100 @@ define(['common/errorDisplay', 'common/format', 'common/html', 'common/ui'], (
      * @returns {boolean} true|false
      */
     function isValidJobInfoObject(jobInfo) {
+        const required = ['app_id', 'app_name', 'batch_id', 'job_id', 'job_params'];
+        try {
+            if (
+                jobInfo !== null &&
+                typeof jobInfo === 'object' &&
+                required.every((key) => key in jobInfo) &&
+                Utils.objectToString(jobInfo.job_params) === 'Array' &&
+                // job_params is empty
+                (jobInfo.job_params.length === 0 ||
+                    // job params is populated with an object
+                    (Utils.objectToString(jobInfo.job_params[0]) === 'Object' &&
```

Comment:
Does it have to be populated? I think there are (very simple) apps that don't require parameters.

Reply:
Fixed in https://github.com/kbase/narrative/pull/2646